api/v2.0/legacy_swagger.yaml

```diff
@@ -3631,12 +3631,12 @@ definitions:
     type: object
     allOf:
       - $ref: '#/definitions/ChartAPIError'
-  BadRequestFormatedError:
+  BadRequestFormattedError:
     description: Bad request
     type: object
     allOf:
       - $ref: '#/definitions/ChartAPIError'
-  ConflictFormatedError:
+  ConflictFormattedError:
```

Comment:
I think this fix is ok, as it just an Object define in swagger, would not break api

Reply:
This has currently fallen out of the PR. I think it might make sense for me to make swagger oriented changes as one PR.

I think what I'll do is once people are ok with a set of changes, I'll make a PR for just that set with a single squashed commit and then recycle this PR to have the next round. The advantage of that is that if there are comments which were already made on a change (e.g. yours and mine here), if we're lucky, they'll stop being outdated once the changes are back (if we're unlucky, they won't, but at least the comments will be present in the PR -- I'm not sure what happens, while I've done a lot of these PRs, I haven't played w/ all possible workflows).